config/core/test-configs.yaml

```diff
@@ -2129,6 +2141,7 @@ test_configs:
     test_plans:
       - baseline
       - baseline-nfs
+      - kselftest-alsa
```

Comment:
This platform has only one instance online so there's not enough capacity for running anything else than baseline. Please drop this line.